Modelica/Media/package.mo

```diff
@@ -4544,6 +4544,11 @@ This function computes an isentropic state transformation:
         min=-1.0e5,
         max=1.e5) "Type for mass flow rate with medium specific attributes";
 
+    replaceable partial function massFraction "Return independent mass fraction"
+      extends Modelica.Icons.Function;
+      input ThermodynamicState state "Thermodynamic state record";
+      output MassFraction Xi[nXi] "Independent mass fraction";
+    end massFraction;
```

Comment:
```suggestion
    replaceable partial function massFraction "Return independent mass fractions"
      extends Modelica.Icons.Function;
      input ThermodynamicState state "Thermodynamic state record";
      output MassFraction Xi[nXi] "Independent mass fractions";
    end massFraction;
```

I noticed that both _mass fraction_ and _mass fraction**s**_ are used for the same thing. Ideally, it should be named consistently.

Reply:
Yes it should be "mass fractions" not "mass fraction". This would then be consistent to all other parameter descriptions in Media. 
(Exception seems to be density_derX but let us leave this for another PR)